CDTDatastoreReplicationAcceptanceTests/ReplicationSettings.plist

```diff
@@ -2,21 +2,21 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>TEST_COUCH_USERNAME</key>
-	<string></string>
-	<key>TEST_COUCH_PASSWORD</key>
-	<string></string>
 	<key>TEST_COUCH_HOST</key>
-	<string>localhost</string>
-	<key>TEST_COUCH_PORT</key>
-	<string>5984</string>
+	<string>cloudantsync002.bristol.uk.ibm.com</string>
```

Comment:
Do these plist changes really need to be checked in, given that it gets modified anyway?
I can see that some of them are perhaps more suitable defaults, but I'd be reluctant for a host name to be checked in.